lightning/backend/local.go

```diff
@@ -1202,7 +1209,23 @@ func (local *local) isIngestRetryable(
 		return false, nil, nil
 	}
 
+	getRegion := func() (*split.RegionInfo, error) {
+		for i := 0; ; i++ {
+			newRegion, err := local.splitCli.GetRegion(ctx, region.Region.GetStartKey())
+			if err != nil {
+				return nil, errors.Trace(err)
+			}
+			if newRegion != nil {
+				return newRegion, nil
+			}
+			log.L().Warn("get region by key return nil, will retry", zap.Reflect("region", region),
+				zap.Int("retry", i))
+			time.Sleep(time.Second)
```

Comment:
select this `time.Sleep` with a `<-ctx.Done()` to avoid Ctrl+C being unresponsive